From: Tim Gardner <tim.gardner@canonical.com>
Date: Thu, 24 Apr 2008 20:08:44 +0000 (-0600)
Subject: UBUNTU: SAUCE: Added support for HDAPS on various ThinkPads from Lenovo and
X-Git-Url: http://kernel.ubuntu.com/git?p=ubuntu%2Fubuntu-intrepid.git;a=commitdiff_plain;h=b42c35c4b137200edfc1e284562fc8c67169ad5e

UBUNTU: SAUCE: Added support for HDAPS on various ThinkPads from Lenovo and
IBM
OriginalAuthor: Klaus S. Madsen <ubuntu@hjernemadsen.org>
OriginalAuthor: Chuck Short <zulcss@ubuntu.com>

This is a combination of Hardy commits
cf0420a9db2c350664b008f6d2c60d914f518d33
25e4d37e811ac124c4afd280f53018c219f9be24
738b43e7c90af839fd09b505045ce70fe21e1cfe
4350133c40a723a4dedd8f94abf2cc658e3b3a80

Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---

Index: 2008-devel/drivers/hwmon/hdaps.c
===================================================================
--- 2008-devel.orig/drivers/hwmon/hdaps.c
+++ 2008-devel/drivers/hwmon/hdaps.c
@@ -525,6 +525,9 @@ static struct dmi_system_id __initdata h
 	HDAPS_DMI_MATCH_NORMAL("IBM", "ThinkPad X41"),
 	HDAPS_DMI_MATCH_INVERT("LENOVO", "ThinkPad X60"),
 	HDAPS_DMI_MATCH_NORMAL("IBM", "ThinkPad Z60m"),
+	HDAPS_DMI_MATCH_NORMAL("IBM", "ThinkPad T61P"),
+	HDAPS_DMI_MATCH_NORMAL("IBM", "ThinkPad R61"),
+	HDAPS_DMI_MATCH_NORMAL("IBM", "ThinkPad T61"),
 	{ .ident = NULL }
 };
 
